### ISSUE TITLE
Multiple series forecast

### ISSUE THREAD

**Issue by JustM57** [facebook/prophet#2350]
It's clear that we can build several models with independent trend, seasonality and coefficients for covariates.
Is it possible to perform forecast for multilpe series with a single model? I mean have different trend seasonality, but shared(same) coefficients for covariates?

**Comment by tcuongd** [comment]
Yeah this has been brought up from time to time, but there's no active work on new modelling techniques at the moment. Some strategies are discussed here to modify the existing prophet model: https://github.com/facebook/prophet/issues/49

`neuralprophet` might have an approach already, although I don't think it uses shared coefficients.
